llvm/lib/Target/AMDGPU/Disassembler/AMDGPUDisassembler.cpp

```diff
@@ -1629,102 +1626,130 @@ static int64_t getInlineImmValBF16(unsigned Imm) {
   }
 }
 
-unsigned AMDGPUDisassembler::getVgprClassId(const OpWidthTy Width) const {
+unsigned AMDGPUDisassembler::getVgprClassId(unsigned Width) const {
```

Comment:
We have such functions elsewhere, so might be possible to combine them.

Reply:
That would be great! There are far too many tables that need updating when you add support for a new register width.

Reply:
Just in case: I'm not planning combining them in this patch.